.changeset/stupid-oranges-tie.md

```diff
@@ -0,0 +1,5 @@
+---
+'@confio/relayer': patch
+---
+
+Setup changesets to keep the changelog up to date.
```

Comment:
The changeset filename is automatically generated.

When a release happens, all the changesets are collected to create a nice release note in the `CHANGELOG.md` and in GitHub releases. Then the changesets are pruned.